mvn-defaults/pom.xml

```diff
@@ -355,17 +355,17 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.jaxrs</groupId>
                 <artifactId>jackson-jaxrs-json-provider</artifactId>
-                <version>2.9.8</version>
+                <version>2.18.1</version>
             </dependency>
             <dependency>
                 <groupId>com.sun.xml.bind</groupId>
                 <artifactId>jaxb-core</artifactId>
-                <version>2.3.0.1</version>
+                <version>4.0.5</version>
```

Comment:
This seems like a big change. Sometimes these changes are not runtime compatible. Can we verify this with downstream docs somewhere, or ensure it with some additional verification.

Reply:
Or even better, remove the dependency chain if it is vestigial.

Reply:
Removed and the build functioned.